### DISCUSSION TITLE
Rajouter une commande !rules

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#782] [Nouvelles idées]
Permettrait de connaitre les règles de base du bot, permettant ainsi de les diffuser à travers le royaume de draftland

**Comment by Pietagorh** [comment]
Avec une petite close "Ces règles ne sont pas absolues, le staff se donne le droit de vous rappeler à l'ordre si vous faites un truc limite"

**Comment by BastLast** [comment]
bonne idée ca

**Comment by BastLast** [comment]
autorisation de dev ca quand les bugs placés dans la milestone 2.2.3 sont fixés
